### DISCUSSION TITLE
Buffering packet

### DISCUSSION THREAD

**Discussion by vuhailongkl97** [bluenviron/mediamtx#2723] [Questions] [answered]
### Question

Dear 
Currently i meet a problem with input source which have very low fps (around 5fps)
I setup to send the stream to mediamtx then got output from mediamtx via vlc the video seem very bad, it wasn't realtime. i expect output stream from mediamtx should around 4-5 fps.
Are there any logic/setting in mediamtx which made the bad result ? 
Thanks.

**Comment by dalamardruid** [comment, marked as answer]
Have you tried reducing VLC's network cache?

https://github.com/bluenviron/mediamtx#vlc

[a comment by an automated account is not shown]
